keyboards/ambergon/multipleglove/keymaps/default/keymap.c

```diff
@@ -0,0 +1,212 @@
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [4] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [5] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [6] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [7] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [8] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [9] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [10] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [11] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [12] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [13] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [14] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    ),
+    [15] = LAYOUT(
+        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
+        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
+        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
+        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
+        KC_Q  ,   KC_R
+    )
```

Comment:
Padding out the keymap array with useless layers is not required.
```suggestion
    [0] = LAYOUT(
        KC_A  ,   KC_B  ,   KC_C  ,   KC_D  ,
        KC_E  ,   KC_F  ,   KC_G  ,   KC_H  ,
        KC_I  ,   KC_J  ,   KC_K  ,   KC_L  ,
        KC_NO ,   KC_NO ,   KC_NO ,   KC_NO ,
        KC_Q  ,   KC_R
    )
```